google-cloud-spanner/src/main/java/com/google/cloud/spanner/Database.java

```diff
@@ -145,7 +145,7 @@ public Page<Operation> listDatabaseOperations() {
 
   /** Returns the IAM {@link Policy} for this database. */
   public Policy getIAMPolicy() {
-    return dbClient.getDatabaseIAMPolicy(instance(), database());
+    return dbClient.getDatabaseIAMPolicy(instance(), database(), 1);
```

Comment:
What is the `1` here?

EDIT: Apparently this is a version number. Should we add the version as an argument to the `getIAMPolicy` method?